constants/nav_links.js

```diff
@@ -1,24 +0,0 @@
-const NAV_LINKS = [
```

Comment:
I'm sorry for the miscommunication, could we put this file back?

Reply:
We put the file back.